src/main/webapp/scheduled-interviews.jsp

```diff
@@ -22,6 +22,8 @@
           <ul class="list-group list-group-flush">
             <li class="list-group-item">Interviewee: ${scheduledInterview.getInterviewee()}</li>
             <li class="list-group-item">Interviewer: ${scheduledInterview.getInterviewer()}</li>
+            <!-- TODO: make sure this looks right after we finish the shadow flow. -->
```

Comment:
I don't know if Jack or I commented on this, but TODOs without attribution (either a bug/github issue or a person's username) attached are generally discouraged. This it to prevent the code getting populated with unactionable todos.

This isn't super important, so I don't necessarily recommend a change here. Just wanted to call out something we appeared to have missed.

Reply:
l agree that the TODOs should be more concrete than just comments floating in the code. l made an issue.